docs.html

```diff
@@ -630,17 +626,19 @@ <h3 class="card-title">Our Chaning Work</h3>
 <section  class="section" >
   <div class="component-title" >Snackbar </div>
   <div class="component-detail" >
-    Snackbars/Toast inform users of a process that an app has performed or will perform. They appear temporarily, towards the bottom of the screen. They shouldn’t interrupt the user experience, and they don’t require user input to disappear.
+     <span  class="highlight-text" >
+      Snackbars/Toast
+    </span>   inform users of a process that an app has performed or will perform. They appear temporarily, towards the bottom of the screen. They shouldn’t interrupt the user experience, and they don’t require user input to disappear.
   </div>
 
   <div id="snackbar"></div>
   <div id="snackbar-center"></div>
```

Comment:
Why are there empty divs? If no use you can remove it

Reply:
This divs are for snackbar they will be visible only one click and for 5 seconds and linked to javascript so nothing to write in `div` element

Reply:
Okay.